build_config.rb

```diff
@@ -8,7 +8,7 @@
     toolchain :gcc
   end
 
-  enable_debug
+  #enable_debug
```

Comment:
OK, I have no opinion which is better, but disabling debug is independent from your method caching code, right?
If so, IMHO you should not include this change in your mcache PR, and make another PR to remove `enable_debug` in build_config.rb.